src/System.Collections.NonGeneric/tests/System.Collections.NonGeneric.Tests.csproj

```diff
@@ -8,6 +8,8 @@
     <RootNamespace>System.Collections.NonGeneric.Tests</RootNamespace>
     <AssemblyName>System.Collections.NonGeneric.Tests</AssemblyName>
     <ProjectGuid>{EE95AE39-845A-42D3-86D0-8065DBE56612}</ProjectGuid>
+    <TargetFrameworkVersion>v5.0</TargetFrameworkVersion>
+    <TargetFrameworkProfile />
```

Comment:
This is probably because the Microsoft.xunit.performance assemblies are built as v5.0 portable assemblies, which forces any referencing assemblies to do the same.